### PR TITLE
simplify "arrange the repeats" user interface

### PR DESCRIPTION
**Pull request by veronika**
Translation arrows are consistent with current behaviour.

Colours:
The colour conventions proposed by @Cowboynoon do not include a blue.  I suppose we could use yellow for the base tile but this is not easy to see on a white background with transparency.  Also note, the black arrows do not show up on a black background and the grey tiles are difficult to see.  More consideration must be given for a reverse video colour scheme.

### CODE REVIEW

**Comment by Cowboynoon** [comment]
apologies, I commented in the wrong place - Github is still very new to me, please excuse my fumbling around:

I copy and paste a previous comment here for relevance:

a note on my suggested yellow - it is more of a khaki - I specifically chose a shade that would not disappear on white, is not incongruous with the international lace diagram code, and to compliment the purple. (Purple-yellow, Green-red) I too had a concern about the blue square,  but I think the "darker yellow" might have a similar tone to the problematic blue square

**Comment by veronika** [comment]
Here is what it looks like in yellow/khaki (instead of blue) with a white and black background:

arrange_opt_white

arrange_opt_black

I have given the nudge arrows a white stroke so that they show up on the black background.  I have also changed the grey squares and the rose ground pattern colours and transparency slightly.

**Comment by jo-pol** [comment]
I thought you wanted to get rid of the circles in the rose ground.

**Comment by veronika** [comment]
I think the pattern helps to understand the overlap but I am also OK with removing it.  I didn't include it in my original mock-up because it takes too long to draw :-)

I also suggested moving the overlap case to the far right because you usually scan left to right and the complexity of the overlap case may overwhelm the user from looking at the rest.

So is the following what we want? 

arrange_opt_white2

arrange_opt_black2

**Comment by Cowboynoon** [comment]
I think it's great :) Clear and concise

As for the colour - I quite like it, it is optimal for a light background, so it's a bit dark on black, but it's a step towards a "brand identity", and while that definitely needs careful consideration, it is probably not the highest priority at the moment. (GF's "dark theme" can one day get it's own custom tones..)

**Comment by jo-pol** [comment]
The yellow brick tiles seem to miss their link

**Comment by jo-pol** [comment]
off topic: clearing the foot side field causes trouble

**Comment by veronika** [comment]
I think the footside problem should be tracked separately.  It has existed for a while and I don't think it is the result of this change.  The footside problem I have seen is that the stitches get mixed up in the prototype (removing footside in prototype should only remove the footside nodes and stitches but internal stitches are somehow affected).  Is this the same footside problem you are seeing?

File: docs/tiles.html [hunk 1]
```diff
@@ -243,552 +243,94 @@ <h2>Forms for advanced users</h2>
...
+                    <rect x="168.27" y="31.08" width="22.675" height="22.675" ry=".24461" style="fill-opacity:.5;fill:#cec511;opacity:1;stroke-linecap:round;stroke-width:1.9481;stroke:#cec511"/>
+                </a>
+                <path d="m30.375 31.165 2e-5 30.141" style="fill:none;stroke-linecap:round;stroke-linejoin:bevel;stroke-width:1.1279;stroke:#029807"/>
+                <rect x="16.735" y="47.665" width="27.28" height="27.28" ry=".2943" style="fill:none;stroke-linecap:round;stroke-width:2.3437;stroke:#029807"/>
+                <rect x="46.885" y="47.665" width="27.28" height="27.28" ry=".2943" style="fill:none;stroke-linecap:round;stroke-width:2.3437;stroke:#834bce"/>
+                <rect x="46.885" y="17.525" width="27.28" height="27.28" ry=".2943" style="fill:none;stroke-linecap:round;stroke-width:2.3437;stroke:#cccccc"/>
+                <rect x="49.63" y="111.02" width="27.529" height="12.134" ry=".1309" style="fill:none;stroke-linecap:round;stroke-width:2.3437;stroke:#834bce"/>
+                <a transform="matrix(2.0094 0 0 2.0094 -525.21 -208.97)" xlink:href="javascript:asHorBricks()">
```
Comment (jo-pol) [inline docs/tiles.html:276]: This seems to activate only the border of the purple brick. Same for the object with `asVerBricks`

Comment (veronika) [reply docs/tiles.html:276]: Are you looking at the code I just submitted?  In the latest submission, the yellow bricks are activated.

Comment (jo-pol) [reply docs/tiles.html:276]: No, I was some 10 minutes behind and just saw you fixed it. I think the screen shots in the tutorial should also change. Then we're done. Perhaps also the colors around the form fields:
https://github.com/d-bl/GroundForge/blob/4b6573e581e48eb9bd5c2c66eb1f945aa4ce51e2/docs/tiles.html#L340-L349

**Comment by veronika** [comment]
Screenshots are in a different repository so they will be under a different pull request.

**Comment by jo-pol** [comment]
The tutorials are not yet in another repository. 

By the way, I don't want to cause too much trouble but for now I personally (matter of taste, not a veto) prefer the bright blue (of the info button) until we are seriously redesigning the branding. Lets reserve the difficult yellow for gimps and tallies. I was a bit too quick publishing the color conventions.

**Comment by veronika** [comment]
Darn!  I wish I had seen your comment about preferring blue over yellow before I updated all of the docs!
I have to run to a meeting so I can't change the yellows to blues right away.  I will look at this after my meeting.

**Comment by veronika** [comment]
Here is what it looks like in blue with a white and a black background:

arrange_opt_blue_white

arrange_opt_blue_black

The blue colour is #0d6efd  (RGB 13, 110, 253).

**Pull request merged**